index.html

```diff
@@ -82,7 +82,7 @@ <h3>Marzoval</h3>
         <div class="container">
 
             <div class="playlist">
-                <div class="video">
+                <section class="video">
                     <div class="content--video">
                         <iframe width="172" height="95" src="https://www.youtube.com/embed/5oZ9epHAdm4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                     </div>
```

Comment:
Please consider using Semantic tags instead of divs where necessary.

Reply:
You did a wonder job commenting your code 👍 but please ensure to always state explicit reasons why you requested for a change; it might be obvious to you and not so for others.